index.js

```diff
@@ -15,7 +15,6 @@
 var createError = require('http-errors')
 var debug = require('debug')('send')
 var deprecate = require('depd')('send')
-var destroy = require('destroy')
```

Comment:
@dougwilson that is correct. I've even added `afterEach` assertions to make sure that all open file descriptors have been closed after every test has run. Although, to be honest, supporting node < 0.10 does make working with streams much more tricky. Node >= 0.10 has the Stream2 implementation and supports the `autoClose` parameter when using `fs.ReadStream`. To support the `autoClose` option I'm exposing in `send()` I have to overwrite the `fs.ReadStream` instance's and `destroy()` method to ensure that it won't close the file descriptor.